src/WebJobs.Extensions.Http/Routing/DefaultWebJobsRouteHandler.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Http.Routing
```

Comment:
I think some of your Builder APIs/extensions were under the parent Http namespace, but this one is under Routing? Is this under the same namespace as the method used to add an instance of this?